internal/driver/snmpdriver.go

```diff
@@ -98,87 +101,87 @@ func (s *SNMPDriver) HandleReadCommands(deviceName string, protocols map[string]
 
 		switch val.(type) {
 		case string:
-			cv, _ := dsModels.NewCommandValue(reqs[i].DeviceResourceName, now, val, dsModels.String)
+			cv, _ := dsModels.NewCommandValue(reqs[i].DeviceResourceName, v2.ValueTypeString, val)
 			res = append(res, cv)
 		case int64:
 			safeVal, ok := val.(int64) // Alt. non panicking version
```

Comment:
in the switch section
`switch safeVal := val.(type) {`
Then, we could remove all type type conversion in the cases.

Reply:
2 exception (`uint` and `default`)